.storybook/main.js

```diff
@@ -1,10 +1,30 @@
+const webpack = require('webpack')
+
 module.exports = {
```

Comment:
I mustve merged a PR that merged next > 12, which uses webpack 5. So i had to update storybook to work with webpack5.

Webpack5 does not include polyfills for certain packages that webpack4 shimmed by default. So this webpack config just shims the right things to get storybook to work again

Reply:
sounds good!